xwiki-platform-core/pom.xml

```diff
@@ -637,6 +637,15 @@
                     "methodName": "getAllFilters",
                     "elementKind": "method",
                     "justification": "Young API"
+                  },
+                  {
+                    "code": "java.method.addedToInterface",
+                    "new": "method void org.xwiki.query.jpql.analysis.Analysis::caseAIntegerComparableItem(org.xwiki.query.jpql.node.AIntegerComparableItem)",
+                    "package": "org.xwiki.query.jpql.analysis",
+                    "classSimpleName": "Analysis",
+                    "methodName": "caseAIntegerComparableItem",
+                    "elementKind": "method",
+                    "justification": "Support for queries of type 'GROUP BY 1' and 'ORDER BY 1'"
```

Comment:
The justification is not correct. It's a justification explaining why backward compatibility should be broken. And we should not break it, unless in very rare cases.

Reply:
I think the rationale is that this doesn't break backward compatibility because generated code from the grammar is not considered public API. 

@All: BTW maybe we have a mistake and this code should be moved to internal package.

Reply:
Yes I don't think this code was really meant to be public.

Reply:
Created https://jira.xwiki.org/browse/XWIKI-15265

Reply:
Ok then I'll remove these lines once we fix the issue that you've created.

Reply:
@atallahade I've just closed https://jira.xwiki.org/browse/XWIKI-15265 :)